### ISSUE TITLE
Adding a new issue to my work.

### ISSUE THREAD

**Issue by gbunn100** [gbunn100/Learning-More-Stuff#1]
Adding steps for resolving an issue
- [ ] Create a branch
- [ ] Add a bio for gbunn100
- [ ] Open a pull request 
- [ ] Get feedback
- [ ] Merge pull request

This will fix #1. This verbiage will automatically close this issue once the request is merged. I can also use the tenses of the word fix or resolve or close.
